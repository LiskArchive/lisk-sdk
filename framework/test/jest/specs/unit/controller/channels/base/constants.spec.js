const {
	INTERNAL_EVENTS,
} = require('../../../../../../../src/controller/channels/base/constants');

describe('base/constants.js', () => {
	it('INTERNAL_EVENTS must match to the snapshot.', () => {
		expect(INTERNAL_EVENTS).toMatchSnapshot();
	});

	it('INTERNAL_EVENTS array should be immutable', () => {
		expect(() => INTERNAL_EVENTS.push('test')).toThrow(TypeError);
		expect(() => INTERNAL_EVENTS.pop()).toThrow(TypeError);
	});
});
