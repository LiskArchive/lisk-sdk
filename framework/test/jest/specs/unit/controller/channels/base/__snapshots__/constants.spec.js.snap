// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`base/constants.js INTERNAL_EVENTS must match to the snapshot. 1`] = `
Array [
  "registeredToBus",
  "loading:started",
  "loading:finished",
]
`;
